public/gameInfo.php

```diff
@@ -343,246 +343,249 @@ function generateGameMetaDescription(
 
 <?php RenderContentStart($pageTitle); ?>
 <?php if ($isFullyFeaturedGame): ?>
-    <script defer src="https://www.gstatic.com/charts/loader.js"></script>
-    <script>
-    document.addEventListener('DOMContentLoaded', function() {
-        if (typeof google !== 'undefined') {
-            google.load('visualization', '1.0', { 'packages': ['corechart'] });
-            google.setOnLoadCallback(drawCharts);
-        }
-    });
+    <?php if ($numDistinctPlayers): ?>
```

Comment:
This (and the closing `endif`) are the relevant lines of code. Everything else is just whitespace adjustment.

Reply:
As a reviewer you can hide whitespace changes by adding a "&w=1" to the URL or clicking the gear between the tabs and the diff and checking the box.